core/src/main/java/org/jruby/RubyHash.java

```diff
@@ -1332,11 +1332,13 @@ public RubyFixnum hash(ThreadContext context) {
         return context.runtime.newFixnum(hval[0]);
     }
 
+    private static final ThreadLocal<byte []> hash16byte = ThreadLocal.withInitial(() -> {return new byte[16];});
```

Comment:
Style issues:

* No space before array declaration: `byte[]`.
* static final constant should be all caps: `HASH16BYTE` or similar

Reply:
@headius thanks